admin/includes/template/javascript/zcJSFramework.js

```diff
@@ -6,110 +6,93 @@
  */
 
 if (typeof zcJS == "undefined" || !zcJS) {
-  window.zcJS = { name: 'zcJS', version: '0.1.0.0' };
-};
+    window.zcJS = {name: 'zcJS', version: '0.1.0.0'};
+}
+;
 
 zcJS.ajax = function (options) {
-  options.url = options.url.replace(/&amp;/g, '&');
-  var deferred = $.Deferred(function (d) {
-      var defaults = {
-          cache: false,
-          type: 'POST',
-          traditional: true,
-          dataType: 'json',
-          data: $.extend(true,{
-            securityToken: securityToken
-        }, options.data)
-      },
-      settings = $.extend(true, {}, defaults, options);
+    options.url = options.url.replace("&amp;", "&");
```

Comment:
I think the reason we'd changed to `(/&amp;/g, '&')` instead of using double-quotes was for the sake of validators misreading the symbols. Would have to look back at history to confirm.